src/GLFW.jl

```diff
@@ -22,8 +22,23 @@ else
 	error("GLFW $VERSION is not supported")
 end
 
+immutable GLFWError <: Exception
+    code::ErrorCode
+    description::UTF8String
+end
+function Base.showerror(io::IO, e::GLFWError)
+    print(io,
+        "GLWF reported an error with the code: ",
```

Comment:
How about the middle ground:
`GLFWError with code: $code, $description`
I think it's good to print all available information...